modules/local/simpleaf_index.nf

```diff
@@ -0,0 +1,48 @@
+process SIMPLEAF_INDEX {
```

Comment:
Does it make sense for this to go into `nf-core/modules`?

Reply:
Hi Greg,
I do think it is. But maybe something we can do after version is released and also after new modules structure is more stable?

Reply:
fair enough, this would unnecessarily delay this release. Let's aim for that in the next one.